lib/user.rb

```diff
@@ -0,0 +1,15 @@
+# Class to make users
+class User
+  attr_reader :name
+
+  attr_writer :marker
+
+  def initialize(name)
+    name += "\'s" if name[-1] != 's'
+    @name = name
+  end
+
+  def mark
+    @marker
+  end
+end
```

Comment:
Nice to have the class in a separated file :+1:

Reply:
We actually resolved it on the moment you were reviewing it.. sorry about that